pgjdbc/src/test/java/org/postgresql/test/jdbc2/ConnectionTest.java

```diff
@@ -261,6 +269,20 @@ public void testTransactionIsolation() throws Exception {
       // Ok.
     }
 
+    try {
+      con.setTransactionIsolation(Connection.TRANSACTION_NONE);
+      fail("TRANSACTION_NONE cannot be used because it specifies that transactions are not supported.");
+    } catch (SQLException e) {
+      // Ok.
+    }
+
+    try {
+      con.setTransactionIsolation(Integer.MIN_VALUE);
+      fail("The given parameter is not one of the allowed Connection constants");
+    } catch (SQLException e) {
+      // Ok.
```

Comment:
KO.
Please verify error code.

Reply:
I will provide a real test, this was just a quick test.